### PR TITLE
27736 (1.4 BSGs): Add hot spare drive identify behavior to TrueNAS guide:

### PR DESCRIPTION
**Pull request by Mrt134**
Change affects ES12, ES24, ES60, X-Series, and M-Series BSGs
Local HTML and PDF build tests: no issues.

### CODE REVIEW

File: userguide/snippets/es24m_drivetrays.rst [hunk 1]
```diff
@@ -4,8 +4,9 @@ Install Drive Trays
 Drive trays are used to mount drives in the array. Drive trays are
 installed in drive bays in the chassis. Each drive bay in the chassis
 has two indicator LEDs to the right of the tray. The status LED is
-blue when the drive is active, and the fault LED is red if a fault has
-occurred.
+blue when the drive is active, the fault LED is red if a fault has
+occurred, and the status and fault LEDs blink when the drive is
```
Comment (wblock) [inline userguide/snippets/es24m_drivetrays.rst]: Should probably s/blink/both blink/, if that's correct.

File: userguide/snippets/es60.rst [hunk 2]
```diff
@@ -191,7 +191,11 @@ the top cover. Slide the top cover forward, then lift it off.
 Install Drive Trays
 ^^^^^^^^^^^^^^^^^^^
 
-Drive trays are used to mount drives in the array.
+Drive trays are used to mount drives in the array. These trays have
+two status lights for drives: green and amber. The drive tray light is
```
Comment (wblock) [inline userguide/snippets/es60.rst]: This kind of switches back and forth between there being two lights and a single one.  Are there really two LEDs, or is it a single dual-color LED?

File: userguide/snippets/es60.rst [hunk 3]
```diff
@@ -191,7 +191,11 @@ the top cover. Slide the top cover forward, then lift it off.
...
+solid green when a drive is connected, solid amber if there is an issue
```
Comment (wblock) [inline userguide/snippets/es60.rst]: Why the separate mention of "issue" and "fault"?  Is there a difference?  Can we remove "issue"?

File: userguide/snippets/es60.rst [hunk 4]
```diff
@@ -191,7 +191,11 @@ the top cover. Slide the top cover forward, then lift it off.
...
+or fault with the drive, or solid green with the amber light blinking
+when the drive is designated as a hot spare.
```
Comment (wblock) [inline userguide/snippets/es60.rst]: s/is designated as/is/

File: userguide/snippets/x_drivetrays.rst [hunk 5]
```diff
@@ -2,8 +2,9 @@ Install Drive Trays
 ~~~~~~~~~~~~~~~~~~~
 
 Drive trays are used to mount drives in the chassis. Each drive tray
-has a status LED which is blue when active or amber if a fault has
-occurred.
+has a status LED which is blue when active, amber if a fault has
+occurred, or solid blue and blinking amber if the drive is designated as
```
Comment (wblock) [inline userguide/snippets/x_drivetrays.rst]: s/is designated as/is/

**Comment by Mrt134** [comment]
Waiting for next round of draft BSGs before merging this.

**Pull request merged**